docs/_static/vcr/vcr_v2.yaml

```diff
@@ -725,6 +725,10 @@ components:
           type: boolean
           description: Indicates whether the Verifiable Credentials within the VP must be verified, default true.
           default: true
+        allowUntrustedIssuer:
+          description: If set to true, trust in credential issuer is not verified. Non did:nuts use cases should set this to true.
```

Comment:
Or derive it from issuer DID? If `did:web`, allowUntrusted = true?

Reply:
That works too, but I don't like magic. The API then has different behavior depending on the did method. We could add it in the documentation, but everyone just skims through the documentation except when it comes to config flags.

Would changing the default value if !did:nuts method in addition to the current work? This provides sane defaults, but also allows overriding them.